src/app/_services/auth.service.ts

```diff
@@ -965,26 +965,25 @@ export class AuthService implements CanActivate {
     return this.http.get(url).toPromise();
   }
 
-  /**
-   * Service call for password reset
-   * @note This call actually changes the password to a random new one (2019-06-06)
-   * @param email account password should be reset on
```

Comment:
Was there a reason for removing the comments for the method?

Reply:
Mostly just because they were no longer accurate :/